### ISSUE TITLE
优先使用本地git信息匹配

### ISSUE THREAD

**Issue by calidion** [hexojs/hexo-deployer-git#163]
在执行hexo deploy时得到如下说明。

由于我有很多不同的git账号，所以一般情况下，我是不会匹配全局账号的。
但是hexo deploy时则会要求全局账号，这种情况可能会影响到不同源码库的提交。
所以建议使用当前库里的github账号。

即在hexo deploy执行的当前目录里寻找 .git/config里面的账号信息进行deploy.
而我的.git/config已经匹配了user的信息

``` sh
*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.
```

**Comment by uiolee** [comment]
Now you can specify the committer in `_config.yml`

https://github.com/hexojs/hexo-deployer-git/blob/880a3c3332d82743521f86737b7d815b128e622b/README.md?plain=1#L31-L32

**Comment by stevenjoezhang** [comment]
This option was added in https://github.com/hexojs/hexo-deployer-git/commit/83bb31c4f4069241c2a2bde8499ca01fcb65f16a on April 17, 2016
And was included in version 0.2.0 https://github.com/hexojs/hexo-deployer-git/commit/498f637d36720bafb2fcc8edcb45ea4b380ecf33 released on June 13, 2016
